Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1068,32 +1068,35 @@ void CClientVehicle::SetModelBlocking(unsigned short usModel, unsigned char ucVa
         m_ucMaxPassengers = CClientVehicleManager::GetMaxPassengerCount(usModel);
 
         // Reset handling to fit the vehicle
-        ushort usHandlingModelID = usModel;
-        if (usHandlingModelID < 400 || usHandlingModelID > 611)
-            usHandlingModelID = m_pModelInfo->GetParentID();
+        if (restoreHandling)
```

Comment:
It looks like the value for `restoreHandling` always depends on `IsLocalEntity()`. Why not just use that here?

Reply:
That's a good point. I'll change that asap :)